doc/api/bidi-engine/bidi-engine.md

```diff
@@ -0,0 +1,84 @@
+## [new] Globalize.bidiEngine( options )
+
+Create Bidi engine instance.
+
+### Parameters
+
+** options **
+
+A JSON object with optional parameters defining Bidi reordering
+Remark. This parameter is optional. If not defined, default values are used. These values can be updated at 
+any moment by calling 'setOptions' method of bidi engine.
+
+**isInputVisual** - ordering scheme of source string, true for 'Visual', false for 'Logical' 
+(defaults to false)  
+**isInputRtl** - text orientation of source string, true for 'right-to-left', false for 'left-to-right', 
+undefined for 'conextual' i.e.direction defined by first strong character of input string (defaults to 'conextual')   
+**isOutputVisual** - ordering scheme of target string, true for 'Visual', false for 'Logical' 
+(defaults to false)  
+**isOutputRtl** - text orientation of target string, true for 'right-to-left', false for 'left-to-right',
+undefined for 'conextual' i.e.direction defined by first strong character of input string (defaults to 'conextual')  
+**isSymmetricSwapping** - whether to replace characters with the "mirrored" property in RTL runs 
+(like brackets) by their mirror-image mapping valid values true/false (defaults to false) 
+	
+### Example
+
+```javascript
+  var options = {
+	isInputVisual: true,
+	isInputRtl: false,
+	isOutputVisual: false,
+	isOutputRtl: true,
+	isSymmetricSwapping: true
+  };
+  var bidiEngine = Globalize.bidiEngine( options );
+
```

Comment:
You should always close the snippets. Add \`\`\` in a new line after `var bindingEngine = ...`